docker-compose.yml

```diff
@@ -1,24 +1,54 @@
 version: "3.9"
 
 services:
+    postgres:
+        container_name: postgres
+        image: postgres
+        environment:
+            POSTGRES_USER: root
+            POSTGRES_PASSWORD: helloworld
+            POSTGRES_DB: zk-groups
+            PGDATA: /data/postgres
+        volumes:
+            - postgres:/data/postgres
+            - ./database/seed.sql:/docker-entrypoint-initdb.d/seed.sql
+        ports:
+            - "5432:5432"
+        restart: unless-stopped
+        command: postgres -c listen_addresses='*'
+        networks:
+            - zk-groups
+
     api:
         container_name: api
         build:
             context: ./
             dockerfile: ./apps/api/Dockerfile
         image: ${DOMAIN}-api:latest
         restart: unless-stopped
-        env_file:
-            - .env
         environment:
             - NODE_ENV=production
-            - DB_NAME="/home/node/api/zk-groups.db"
-        volumes:
-            - sqlite:/home/node/api/zk-groups.db
+            - DB_TYPE=postgres
+            - DB_URL=postgres://root:helloworld@postgres:5432/zk-groups
+            - JWT_SECRET_KEY=zk_groups_jwt_secret
```

Comment:
Why de we need to copy all the env variables here? Shouldn't be automatically loaded from the `.env` file?

Reply:
I specified it explicitly here for more visibility - mainly for Devops team. It not safe to load the secrets from .env file nor from docker-compose. We wont have .env file when the repo is cloned in EC2.

We can check with Ops team what would be the best way to handle secret envs. In Kubernetes there are ways to do this, not sure with our proposed setup. For now it was added here so docker-compose will have all the info needed for deployment. Can revert it if this is not helpful.

Reply:
Ahhn I see. It makes sense then!

Reply:
We should just remember to sync the env variables every time we change something